lib/charty/backends.rb

```diff
@@ -0,0 +1,32 @@
+module Charty
+  class BackendNotLoadedError < RuntimeError; end
+
+  module Backends
+    def self.register_backend(backend_class)
+      @backends ||= {}
+      key = backend_class.name[/(?:::)?(\w+)\z/, 1]
+      key.gsub!(/\A([A-Z])/) { $1.downcase }
+      key.gsub!(/([a-z\d])([A-Z])/, '\1_\2')
+      @backends[key] = backend_class
+    end
+
+    def self.create(backend_name)
```

Comment:
We use "finding class API" instead of "creating object API" in table adapters. How about using the same APi in backends?

Reply:
Indeed.  I'll do, too.

Reply:
Done